beets/autotag/mb.py

```diff
@@ -223,6 +223,9 @@ def track_info(recording, index=None, medium=None, medium_index=None,
     if recording.get('length'):
         info.length = int(recording['length']) / (1000.0)
 
+    if recording.get('disambiguation'):
+        info.trackdisambig = recording.get('disambiguation')
```

Comment:
This can be `recording['disambiguation']` because it's guaranteed to exist at this point, I believe.